extensions/gitpod/src/gitpodServer.ts

```diff
@@ -0,0 +1,182 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Gitpod. All rights reserved.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { Log } from './common/logger';
+import fetch from 'node-fetch';
+import { PromiseAdapter, promiseFromEvent } from './common/utils';
+import { withServerApi } from './internalApi';
+import pkceChallenge from 'pkce-challenge';
+import { v4 as uuid } from 'uuid';
+import { Disposable } from './common/dispose';
+
+export const baseURL = 'https://jp-oauth-test.staging.gitpod-dev.com';
```

Comment:
it should be configurable via `gitpod.host`, i.e. `https://${gitpod.host}` on user level

Reply:
what do you mean another another `configuration` in package.json?

Reply:
yes

Reply:
Added `gitpod.serviceUrl` configuration